testnets/lavatestnet2/chain.json

```diff
@@ -56,91 +56,9 @@
     ],
     "genesis": {
       "name": "v0.3.0",
-      "genesis_url": "https://raw.githubusercontent.com/lavanet/lava-config/main/testnet-1/genesis_json/genesis.json"
+      "genesis_url": "https://raw.githubusercontent.com/lavanet/lava-config/main/testnet-2/genesis_json/genesis.json"
     },
     "versions": [
```

Comment:
Please add the current version (v0.32.0) to the `versions` [] array

Reply:
thanks, done